src/components/UploadSong.css

```diff
@@ -0,0 +1,60 @@
+.boxStyle {
+    display: flex;
+    width: 100vw;
+    align-items: center;
+    justify-content: center;
+
+    height: 90vh;
+    font-size: 12px;
+    max-width: 100%;
+    background: #050516;
+}
+
+.uploadFunctionalityStyle {
+    display: flex;
+    width: 45%;
+    height: 40%;
+    padding: 0.5em 1em;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+
+    /* font-weight: 900; */
+    border: dashed 1.1px #ffffff;
+    border-radius: 5px;
+    border-width: 1.5px;
+
+    opacity: 0.5;
+}
+
+.iconStyle {
+    background: none;
+    border: none;
+    outline: none;
+    display: flex;
+    cursor: pointer;
+    margin-bottom: 20px;
+}
+
+.uploadText {
+    display: flex;
+    font-family: Biryani;
```

Comment:
font-family is already added in global index.scss. Why did you need to add it here? Also font-style normal is by default.